docs/reference/commandline/cp.md

```diff
@@ -34,12 +35,12 @@ supplying the initial forward slash is optional; The command sees
 `compassionate_darwin:tmp/foo/myfile.txt` as identical. If a `LOCALPATH` value
 is not absolute, is it considered relative to the current working directory.
 
-Behavior is similar to the common Unix utility `cp -a` in that directories are
-copied recursively with permissions preserved if possible. Ownership is set to
-the user and primary group on the receiving end of the transfer. For example,
-files copied to a container will be created with `UID:GID` of the root user.
-Files copied to the local machine will be created with the `UID:GID` of the
-user which invoked the `docker cp` command.
+Behavior is similar to the common Unix utility `cp -a` (unless `-L` option is
```

Comment:
The parenthetical throws doubt.  What happens if the command encounters a symbolic link. The comments in the code makes it sound like the command follows the link and copies it over with the rest of the contents from the source to the destination.  That's just a guess --> we need to describe it exactly.  If the link is to a file or to a directory does that make a diff?

```markdown
The `cp` command behaves like Unix utility `cp -a` in that directories are copied recursively with permissions preserved if possible. Ownership is set to the user and primary group at the destination. For example, files copied to a container are created with `UID:GID` of the root user. Files copied to the local machine are created with the `UID:GID` of the user which invoked the `docker cp` command.  If you specify the `-L` option, `docker cp` follows any symbolic link in the path...
```

---> preview

The `cp` command behaves like Unix utility `cp -a` in that directories are copied recursively with permissions preserved if possible. Ownership is set to the user and primary group at the destination. For example, files copied to a container are created with `UID:GID` of the root user. Files copied to the local machine are created with the `UID:GID` of the user which invoked the `docker cp` command.  If you specify the `-L` option, `docker cp` follows any symbolic link in the path...